src/components/browserSupport/components/OutdatedOS.js

```diff
@@ -0,0 +1,23 @@
+// libraries
+import React from 'react'
+
+// components
+import ExplanationDialog from '../../common/dialogs/ExplanationDialog'
+
+// assets
+import illustration from '../../../assets/UnsuportedBrowser.svg'
+
+export default ({ onDismiss }) => (
+  <ExplanationDialog
+    title={'Oops! Your iOS is outdated'}
```

Comment:
ask @patpedrosa regarding this message. also show her the screenshots
maybe we would need design adjustements

also we not showing the minimal version required

Reply:
**Oops! Your iOS is outdated**

Please update your iOS to access GoodDollar.
Minimum version required: iOS 12.

[OK, GOT IT]